tendermint/Cargo.toml

```
@@ -55,6 +55,7 @@ thiserror = "1"
 toml = { version = "0.5" }
 uuid = { version = "0.8", default-features = false }
 zeroize = { version = "1.1", features = ["zeroize_derive"] }
+ripemd160 = "0.8.0"
```

This should probably be changed to
```suggestion
ripemd160 = "0.8"
```

---

assumed because others in file were only specified up to minor version that it was the preference, updated